src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -276,6 +276,8 @@ typedef struct uct_ib_mlx5_md {
 #endif
     /* The maximum number of outstanding RDMA Read/Atomic operations per DC QP. */
     uint8_t                  max_rd_atomic_dc;
+    struct ibv_mr*           flush_remote_mr;
+    struct mlx5dv_devx_obj   *flush_remote_devx_mr;
```

Comment:
I think it better be under `#if HAVE_DEVX`

Reply:
> I think it better be under `#if HAVE_DEVX`

ok, moved under `HAVE_DEVX` macro